byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Rule/BHead.hs

```diff
@@ -20,6 +20,8 @@ instance STS BHEAD where
   type Environment BHEAD
     = ( Bimap VKeyGenesis VKey
       , Slot
+      , BlockCount -- this is deemed to be a global constant that we
```

Comment:
What about: "this is a global constant in the formal specification, which we put in this environment so that we can test with different values of it. " Or something like that ...

Reply:
Sounds great! I updated the comment.